doc/advanced.rst

```diff
@@ -383,3 +383,16 @@ the ``sphinx_gallery.gen_gallery`` extension, you can use in ``conf.py``::
     extensions = ['sphinx_gallery.load_style']
 
 This will only cause the ``gallery.css`` file to be added to your build.
+
+Add your own CSS styling
```

Comment:
CSS is already discussed two places in `configuration.rst`, first in tweaking thumbnails then later with mini galleries:

https://sphinx-gallery.github.io/stable/configuration.html

So the content in this PR should probably be worked in there somewhere.